src/controllers/hid/hidcontroller.cpp

```diff
@@ -298,6 +298,34 @@ void HidController::sendBytesReport(QByteArray data, unsigned int reportID) {
     }
 }
 
+void HidController::sendFeatureReport(
+        const QList<int>& dataList, unsigned int reportID) {
+    QByteArray dataArray;
+    for (const int datum: dataList) {
+        dataArray.append(datum);
+    }
+	
+    // Append the Report ID to the beginning of dataArray[] per the API..
+    dataArray.prepend(reportID);
+
+    int result = hid_send_feature_report(
+            m_pHidDevice, (unsigned char*)dataArray.constData(), dataArray.size());
+    if (result == -1) {
+        if (ControllerDebug::enabled()) {
+            qWarning() << "Unable to send data to" << getName()
+                       << "serial #" << hid_serial << ":"
+                       << safeDecodeWideString(hid_error(m_pHidDevice), 512);
+        } else {
+            qWarning() << "Unable to send data to" << getName() << ":"
+                       << safeDecodeWideString(hid_error(m_pHidDevice), 512);
+        }
```

Comment:
I prefer:

```
if (error case) {
  // handle error;
  return;
}

// success case.
```

This reduces the amount of if/else indentation.

Reply:
Done